### PR TITLE
Add Page API Controller

### PR DESCRIPTION
**Pull request by d-a-v-e**
## Changes in this PR

API endpoint ready for contenful Page webhooks

### CODE REVIEW

**Comment by d-a-v-e** [comment]
[Add Page API Controller](https://trello.com/c/UNbrVSQn/33-add-page-api-controller)

File: config/routes.rb [hunk 1]
```diff
@@ -33,9 +33,15 @@
...
   # Contentful
-  post "/api/contentful/auth" => "api/contentful/base#auth"
-  post "/api/contentful/entry_updated" => "api/contentful/entries#changed"
-  post "/api/contentful/category" => "api/contentful/categories#changed"
+  namespace :api do
+    namespace :contentful do
+      post "auth" => "base#auth"
+      post "entry_updated" => "entries#changed"
+      post "category" => "categories#changed"
+      post "page" => "pages#create"
```
Comment (peterdavidhamilton) [inline config/routes.rb:41]: Would `resources :pages, only: %i[create destroy]` work here?

Comment (d-a-v-e) [reply config/routes.rb]: DELETE would require `/:id` although @EdwinKruglov can we do this in contentful?

Screenshot 2021-11-18 at 17 07 56

Comment (EdwinKruglov) [reply config/routes.rb]: @d-a-v-e yes, that works in Contentful.

**Review by PiperS52 — APPROVED** [review]

**Review by EdwinKruglov — APPROVED** [review]

**Pull request merged**